python/GafferSceneUI/PrimitiveInspector.py

```diff
@@ -260,90 +266,115 @@ def __plugParentChanged( self, plug, oldParent ) :
 		# next suitable plug from the current node set.
 		self._updateFromSet()
 
+	# Needed by GafferUI.BackgroundMethod
+	def plug( self ):
+		if self.__scenePlug:
+			return self.__scenePlug["object"]
+		else:
+			return None
```

Comment:
We can avoid making this public using the same mechanism used for associating the pre and post calls with the background method, like so :

```
@__backgroundUpdate.plug
def __backgroundUpdatePlug( self ) :
...
```

Reply:
Ah, cool.  Did it this way.